astropy/table/tests/test_mixin.py

```diff
@@ -124,6 +124,9 @@ def test_io_quantity_write(tmpdir):
             assert isinstance(qt['a'], u.Quantity)
             assert qt['a'].unit == 'Angstrom'
             continue
+        if fmt == 'votable':
+            t.write(filename, format=fmt, overwrite=True)
```

Comment:
The test is not very convincing. There is no check whether it is written correctly or not.